public/src/components/forms/GroupForm.tsx

```diff
@@ -224,11 +223,12 @@ export class GroupForm extends React.Component<IGroupProps, IGroupState> {
 
     private handleSearch(query: string): void {
         query = query.toLowerCase();
-        const filteredData: IUserRelation[] = [];
+        const filteredData: Enrollment[] = [];
         this.props.freeStudents.forEach((student) => {
-            if ((student.user.getName().toLowerCase().indexOf(query) !== -1
-                || student.user.getEmail().indexOf(query) !== -1
-                || student.user.getLogin().indexOf(query)) !== -1
+            const enr = student.toObject().user;
```

Comment:
It is confusing that you use `student` for `Enrollment` type and `enr` for `User` type. I guess also `freeStudents` is a bit confusing, since that's really a list of enrollments. Maybe `availableEnrollments` is a good name?

Reply:
Well, now we work with enrollments here, but we still kinda add users/students (not enrollments) to groups. So naming is still based on the logical UI components, and not underlying structures. Easy to change if you think it makes more sense.
Totally agree about `enr` for `User`, fixing it.